db/c.cc

```diff
@@ -986,6 +986,33 @@ rocksdb_column_family_handle_t* rocksdb_create_column_family(
   return handle;
 }
 
+rocksdb_column_family_handle_t** rocksdb_create_column_families(
+    rocksdb_t* db,
+    const rocksdb_options_t* column_family_options,
+    int num_column_families, const char* const* column_family_names,
+    char** errptr) {
+  std::vector<ColumnFamilyHandle*> handles;
+  std::vector<std::string> names;
+  for (int i = 0; i != num_column_families; ++i) {
+    names.push_back(std::string(column_family_names[i]));
+  }
+  SaveError(errptr,
+      db->rep->CreateColumnFamilies(ColumnFamilyOptions(column_family_options->rep),
+        names, &handles));
+
+  rocksdb_column_family_handle_t** c_handles = static_cast<rocksdb_column_family_handle_t**>(malloc(sizeof(rocksdb_column_family_handle_t*) * num_column_families));
```

Comment:
In this code base, a line is 80-char maximum. Please wrap those lines that are longer.

Reply:
Fixed